libs/deprecations.liq

```diff
@@ -6,7 +6,8 @@
 # @param old Old function name.
 # @param new New function name.
 def deprecated(old, new)
-  log.severe(label="lang.deprecated", "WARNING: \"#{old}\" is deprecated and will be removed in future version. Please use \"#{new}\" instead.")
+  new = if new == "" then "" else " Please use \"#{new}\" instead." end
```

Comment:
nit: `string.cat` is nice way for doing this.. 🙂

Reply:
Not sure I understand this one...

Reply:
```ruby
tail = if new == "" then [] else ["Please use \"#{new}\" instead."]

msg = list.add(
  "WARNING: \"#{old}\" is deprecated and will be removed in future version.",
  tail
)

log.severe(label="lang.deprecated", string.concat(separator=" ", msg))
```

Reply:
Is this really more clear?? :)